src/AsyncMysql/AsyncMysqlQueryErrorResult.hack

```diff
@@ -0,0 +1,20 @@
+namespace Slack\SQLFake;
+
+<<__MockClass>>
+final class AsyncMysqlQueryErrorResult extends \AsyncMysqlQueryErrorResult {
+
+	/* HH_IGNORE_ERROR[3012] I don't want to call parent::construct */
+	public function __construct(private int $mysql_errno = 1105, private string $mysql_error = 'ERUnknownError') {}
+
```

Comment:
think we can get away with calling the parent constructor here and removing the ignore:

```
parent::__construct();
```

Reply:
I don't think we can do that here because the parent constructor is private

Reply:
We can't do that here because the parent constructor is a private method ```[FATAL] E_FATAL: Call to private method AsyncMysqlQueryErrorResult::__construct()```

Reply:
😢